src/test/java/com/google/api/codegen/testdata/discoveries/mvvm_java/java_adexchangebuyer.v1.4.json.baseline

```diff
@@ -39,12 +34,12 @@ public class AdExchangeBuyerExample {
   }
 
   public AdExchangeBuyer createService() throws IOException, GeneralSecurityException {
-    GoogleCredential credential = GoogleCredential.getApplicationDefault();
-
-    if (credential.createScopedRequired()) {
-      credential =
-          credential.createScoped(Arrays.asList("https://www.googleapis.com/auth/cloud-platform"));
-    }
+    // TODO: Change placeholder below to generate authentication credentials.
+    // See: https://foo.com/bar
```

Comment:
It's passed as an option to the generator, and should link to language specific documentation on the auth mechanism